Rome-Blockchain-Labs/terminal-library
# Fix - prevent eager connect

- Bump to v0.1.8
- Remove useLocalStorage as we dont want wallet connection to persist
- Remove eagerconnect logic in useeffect and only activate network connection onload
- Add  getAddChainParametersFromNetworkName

To test:
1. `cd example && yarn && yarn start` 
2. Connect to metamask
3. Refresh page
4. Connect to wallet account should appear again
